dGame/dBehaviors/SpeedBehavior.h

```diff
@@ -13,6 +13,8 @@ class SpeedBehavior final : public Behavior
 
 	void Handle(BehaviorContext* context, RakNet::BitStream* bitStream, BehaviorBranchContext branch) override;
 
+	void Calculate(BehaviorContext* context, RakNet::BitStream* bitStream, BehaviorBranchContext branch) override;
```

Comment:
How was Calculate tested for functionality?

Reply:
this was tested with the pirate parrot crash script, which is what originally caused me to look into this since it's supposed to slow the player down when sneaking past the parrot guards to the pirate treasure. and then speed the player back up when exiting that area.